tests/nodes/draw/test_pose.py

```diff
@@ -139,10 +139,8 @@ def test_if_configs_are_used(
 
         keypoints = np.random.rand(1, 17, 2)  # keypoints shape for 1 person
         keypoint_scores = np.random.rand(1, 17)  # keypoint_scores shape for 1 person
-        keypoint_conns = np.random.rand(
-            1, 15, 2, 2
-        )  # keypoint_conns shape for 1 person
-
+        # keypoint_conns shape for 1 person, cast to object
+        keypoint_conns = np.random.rand(1, 15, 2, 2).astype(object)
```

Comment:
Is there a reason to cast this as `object` while the above `generaly.py:57` is cast as `float`?
I've changed `object` to `float` and the unit test still passes.

Reply:
The only reason to change this object is to ensure that the mocked input `keypoint_conns` has the same type as what the actual output (from a pose estimation model) will be.

The bug addressed in this PR was uncaught because we're mocking a "nice" input for `draw.pose` instead.

If we were to revert the change in `project_points_onto_original_image` and run this test case now, it will fail expected, reproducing the bug experience when running the default pipeline config.

Reply:
In this case, it'll be good to document this reason into the unit test file, so that the purpose of `astype(object)` becomes clear.

Reply:
Added explanation for casting to `dtype=object`